<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplatePage</class>
 <widget class="QWidget" name="TemplatePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTreeView" name="treeView">
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="extractButton">
       <property name="text">
        <string comment="@action:button">Clone Template</string>
       </property>
       <property name="icon">
        <iconset theme="archive-extract"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
